src/leihs/my/language.clj

```diff
@@ -2,18 +2,18 @@
   (:require
     [leihs.core.locale :refer [set-language-cookie]]
     [leihs.my.paths :refer [path]]
-    [clojure.java.jdbc :as jdbc]
+    [next.jdbc :as jdbc]
+    [honey.sql :refer [format] :rename {format sql-format}]
     [compojure.core :as cpj]
     [clojure.tools.logging :as log]
     [ring.util.response :refer [redirect]]))
 
 (defn redirect-back-with-language-cookie
-  [{tx :tx
+  [{tx :tx-next
     {locale :locale} :form-params
-    {referer "referer"} :headers
-    :as request}]
+    {referer "referer"} :headers}]
   (-> (redirect referer)
-      (set-language-cookie (jdbc/get-by-id tx :languages locale :locale))))
+    (set-language-cookie (jdbc/execute-one! tx (sql-format {:select [:*] :from [:languages] :where [:= :locale locale]})))))
```

Comment:
prefer the honeysql functions representation anyway, otherwise each keyword on a newline.

and `->>` ;-)